pyproject.toml

```diff
@@ -1,3 +1,85 @@
+[tool.poetry]
+name = "rasa"
+version = "1.6.2b1"
+description = "Open source machine learning framework to automate text- and voice-based conversations: NLU, dialogue management, connect to Slack, Facebook, and more - Create chatbots and voice assistants"
+authors = ["Rasa Technologies GmbH <hi@rasa.com>"]
+
+[tool.poetry.dependencies]
```

Comment:
as raised earlier: we shouldn't pin the dependencies to patch versions but rather allow ranges. travis builds should check against most recent versions (maybe we also need a build to check against lower boundaries but not sure if that is easily possible)